doc/source/whatsnew/v0.24.0.rst

```diff
@@ -403,6 +403,7 @@ Other Enhancements
 - :meth:`pandas.api.types.is_list_like` has gained a keyword ``allow_sets`` which is ``True`` by default; if ``False``,
   all instances of ``set`` will not be considered "list-like" anymore (:issue:`23061`)
 - :meth:`Index.to_frame` now supports overriding column name(s) (:issue:`22580`).
+- :meth:`Categorical.from_codes` now can take a dtype parameter (:issue:`24398`).
```

Comment:
use double backticks around dtype; say this is an alternative to passing categories/ordered